app/views/maintenance_tasks/runs/_info.html.erb

```diff
@@ -11,6 +11,14 @@
 
 <% if run.csv_file.present? %>
   <div class="block">
-    <%= link_to('Download CSV', csv_file_download_path(run)) %>
+    <%= link_to("Download CSV", csv_file_download_path(run)) %>
+  </div>
+<% end %>
+<% if run.arguments.present? %>
+  <div class="block">
+    <h5>Arguments:</h5>
+    <% run.arguments.each do |key, value| %>
+      <p><%= "#{key}: #{value}" %></p>
```

Comment:
Because we take in arguments via `text_area` inputs, the values are always stored as strings (with users being expected to parse that string into a complex type like a hash / array from within their Task code), so we should be okay here!

Reply:
EDIT: Clarification of my previous comment - since we're using `ActiveModel::Attributes` to power parameters, any `ActiveModel::Type::Value` can be used (and will be stored), it's just that complex types such as arrays / nested hashes will be strings. Although I suppose users could write their own custom type and use it for an attribute, so maybe we do ought to use `inspect` to be safe 🤔